pkgs/development/libraries/aws-sdk-cpp/default.nix

```diff
@@ -28,7 +29,9 @@ in stdenv.mkDerivation rec {
   outputs = [ "out" "dev" ];
   separateDebugInfo = stdenv.isLinux;
 
-  buildInputs = [ cmake curl ];
+  buildInputs = [ cmake curl ]
+    ++ lib.optionals stdenv.isDarwin
+         (with darwin.apple_sdk.frameworks; [ CoreAudio AudioToolbox ]);
```

Comment:
I suppose we could do this, or just conditionally disable those SDKs. I think it's for Amazon Lex and Polly or something like that.

Reply:
We should probably avoid depending on frameworks for this, since it's a dependency of nix.

Reply:
Yeah, that was my concern, although Nix only builds the S3 subset of the SDK (so we could perhaps only enable the frameworks if someone wants all sub-SDKs)

Reply:
I think it's `aws-sdk-cpp.override { apis = ["text-to-speach"]; }` that needs the frameworks.